torch/csrc/distributed/c10d/TCPStore.cpp

```diff
@@ -181,7 +183,11 @@ class SendBuffer {
  public:
   SendBuffer(detail::TCPClient& client, detail::QueryType cmd)
       : client(client) {
-    buffer.reserve(32); // enough for most commands
+    const uint8_t* tmp_ptr = (const uint8_t*)&queryMagicNumber;
```

Comment:
nit: being pedantic here but this could be `auto *tmp_ptr = static_cast<const uint8_t>(queryMagicNumber)`

Also probably not high prio, but if the host system is big endian, this will put the magic number in reverse order.